package.json

```diff
@@ -88,6 +87,7 @@
     "start-server-and-test": "^1.11.0",
     "styled-components": "^4.2.0",
     "typescript": "^3.8.3",
+    "uniswap-xdai-sdk": "3.0.3",
```

Comment:
`uniswap-xdai-sdk` is both in `devDependencies and dependencies`. Which is it? And which version? `3.0.3`?

Reply:
latest